parser/ParseStatement.hpp

```diff
@@ -245,8 +279,8 @@ class ParseStatementCreateIndex : public ParseStatement {
      *
      * @return The list of attributes on which index is to be built.
      **/
-    const PtrList<ParseString>& attribute_name_list() const {
-      return *attribute_name_list_;
+    const PtrList<ParseAttribute>* attribute_list() const {
```

Comment:
Why change the return value from a reference to a pointer? Could `attribute_list_` be `nullptr`?